op-challenger/game/fault/contracts/delayed_weth.go

```diff
@@ -0,0 +1,61 @@
+package contracts
+
+import (
+	"context"
+	"fmt"
+	"math/big"
+
+	"github.com/ethereum-optimism/optimism/op-bindings/bindings"
+	"github.com/ethereum-optimism/optimism/op-challenger/game/fault/contracts/metrics"
+	"github.com/ethereum-optimism/optimism/op-service/sources/batching"
+	"github.com/ethereum-optimism/optimism/op-service/sources/batching/rpcblock"
+	"github.com/ethereum/go-ethereum/common"
+)
+
+var (
+	methodWithdrawals = "withdrawals"
+)
+
+type DelayedWethContract struct {
+	metrics     metrics.ContractMetricer
+	multiCaller *batching.MultiCaller
+	contract    *batching.BoundContract
+}
+
+type WithdrawalRequest struct {
+	Timestamp *big.Int
+	Amount    *big.Int
+}
+
+func NewDelayedWethContract(metrics metrics.ContractMetricer, addr common.Address, caller *batching.MultiCaller) (*DelayedWethContract, error) {
+	contractAbi, err := bindings.DelayedWETHMetaData.GetAbi()
+	if err != nil {
+		return nil, fmt.Errorf("failed to load delayed weth ABI: %w", err)
+	}
+	return &DelayedWethContract{
+		metrics:     metrics,
+		multiCaller: caller,
+		contract:    batching.NewBoundContract(contractAbi, addr),
+	}, nil
+}
+
+// GetWithdrawals returns all withdrawals made from the contract since the given block.
+func (d *DelayedWethContract) GetWithdrawals(ctx context.Context, block rpcblock.Block, gameAddr common.Address, recipients ...common.Address) ([]*WithdrawalRequest, error) {
+	defer d.metrics.StartContractRequest("GetWithdrawals")()
+	calls := make([]batching.Call, 0, len(recipients))
+	for _, recipient := range recipients {
+		calls = append(calls, d.contract.Call(methodWithdrawals, gameAddr, recipient))
+	}
+	results, err := d.multiCaller.Call(ctx, block, calls...)
+	if err != nil {
+		return nil, fmt.Errorf("failed to fetch withdrawals: %w", err)
+	}
+	withdrawals := make([]*WithdrawalRequest, len(recipients))
+	for i, result := range results {
+		withdrawals[i] = &WithdrawalRequest{
+			Timestamp: result.GetBigInt(0),
```

Comment:
Timestamp should be the second value and Amount the first. 
https://github.com/ethereum-optimism/optimism/blob/0b7b81c3e81f7a8903a84742ed1fcea43467d93e/packages/contracts-bedrock/src/dispute/interfaces/IDelayedWETH.sol#L10C1-L13C1

Reply:
Ah yes good catch :)